xml/storage_multipath.xml

```diff
@@ -284,51 +321,144 @@
    </itemizedlist>
   </sect2>
 
+  <sect2 xml:id="sec-multipath-planning-types">
+    <title>Multipath installation types and the initramfs</title>
+    <sect3 xml:id="sec-multipath-planning-type-root">
+      <title>Root file system on multipath (SAN-boot)</title>
+      <para>
+	The root file system is on a multipathed device (usually,
+	all other file systems are on multipathed storage as well).
+	This is typically the case for
+	diskless servers that use SAN or NAS storage exclusively. On such
+	systems, multipath support is required for booting, and multipathing
+	must be enabled in the initramfs (initrd).
+	See <xref linkend="sec-multipath-initrd-sync" xrefstyle="SectTitleOnPage"/>.
+      </para>
+    </sect3>
+    <sect3 xml:id="sec-multipath-planning-type-noroot">
+      <title>Root file system on normal disk</title>
+      <para>
+	The root file system (and possibly some other file systems) are on local
+	storage, e.g. on a directly attached SATA disk or local RAID,
+	but the system additionally uses filesystems on multipathed SAN or NAS
+	storage. This system type can be configured in three different ways:
+      </para>
+      <orderedlist>
+	<listitem>
+	  <para>
+	  Using a root-on-multipath setup. That is, all
+	  block devices are part of multipath maps, including the local disk,
+	  which will appear as a degraded multipath map with just one path.
+	  This configuration results if multipathing was enabled during
+	  initial system installation with &yast;.
+	  It is the simplest configuration, but has a performance
+	  overhead.
```

Comment:
I don't know. Nobody has ever benchmarked this systematically. It would be very easy to make false claims here. I'd rather be vague for the time being.